src/test/java/org/hobbit/core/components/dummy/DummyDataCreator.java

```diff
@@ -34,6 +34,7 @@ protected void generateData() throws Exception {
         byte data[];
         for (int i = 0; i < dataSize; ++i) {
             data = RabbitMQUtils.writeString(Integer.toString(i));
+            
```

Comment:
Please revert changes like that which do not add any functionality but still could lead to merge conflicts, etc.